test/rules/no-duplicates.spec.ts

```diff
@@ -108,9 +108,8 @@ ruleTester.run('no-duplicates', rule, {
 
     // #86: duplicate unresolved modules should be flagged
     test({
-      code: "import foo from 'non-existent'; import bar from 'non-existent';",
       // Autofix bail because of different default import names.
-      output: "import foo from 'non-existent'; import bar from 'non-existent';",
```

Comment:
`output` now implies an autofix, so when nothing changes it needs to be omitted

Reply:
Looks good!